src/board.rs

```diff
@@ -326,6 +322,8 @@ impl Board {
         }
         self.add_castle_moves();
     }
+
+    pub fn replay(&mut self) {}
```

Comment:
I think this should be in game or some other class we create later to deal with tauri.

Reply:
Oh I meant to remove that.